src/Planners/tests/TimeVaryingDCMPlannerTest.cpp

```diff
@@ -23,42 +21,36 @@ TEST_CASE("TimeVaryingDCMPlanner")
 
     ContactListMap contactListMap;
 
-    iDynTree::Transform leftTransform = iDynTree::Transform::Identity();
-    iDynTree::Position leftPosition;
-
-    leftPosition[0] = 0;
-    leftPosition[1] = -0.8;
-    leftPosition[2] = 0;
-    leftTransform.setPosition(leftPosition);
+    // left foot
+    // first footstep
+    manif::SE3d leftTransform = manif::SE3d::Identity();
+    leftTransform.coeffs().head<3>() << 0, -0.8, 0;
     REQUIRE(contactListMap["left"].addContact(leftTransform, 0.0, 1.0));
 
-    leftPosition[0] = 0.25;
-    leftPosition[2] = 0.2;
-    leftTransform.setPosition(leftPosition);
+    // second footstep
+    leftTransform.coeffs()[0] = 0.25;
+    leftTransform.coeffs()[2] = 0.2;
     REQUIRE(contactListMap["left"].addContact(leftTransform, 2.0, 7.0));
 
-    iDynTree::Transform rightTransform = iDynTree::Transform::Identity();
-    iDynTree::Position rightPosition;
-
-    rightPosition[0] = 0;
-    rightPosition[1] = 0.8;
-    rightPosition[2] = 0;
-    rightTransform.setPosition(rightPosition);
+    // right foot
+    // first footstep
+    manif::SE3d rightTransform = manif::SE3d::Identity();
+    leftTransform.coeffs().head<3>() << 0, 0.8, 0;
```

Comment:
same as above and for other coeff() related lines.

Reply:
Also, shouldn't it be 
```suggestion
    rightTransform.coeffs().head<3>() << 0, 0.8, 0;
```
?

Reply:
@S-Dafarra 
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/82/commits/e2e3bc36514b4b761a0ba1d8734473de3694c36a